lib/screens_web/controllers/v2/screen_controller.ex

```diff
@@ -81,4 +92,34 @@ defmodule ScreensWeb.V2.ScreenController do
     |> put_view(ScreensWeb.ErrorView)
     |> render("404.html")
   end
+
+  defp calculate_refresh_rate_offset(screen_id, refresh_rate) do
+    screen_id
+    |> Base.encode16()
+    |> String.to_integer(16)
+    |> rem(refresh_rate)
+  end
+
+  defp screen_ids(target_app_id, refresh_rate) do
+    ids =
+      for {screen_id, %Screen{app_id: ^target_app_id}} <- State.screens() do
+        screen_id
+      end
```

Comment:
Is this for loop effectively the same as using `Enum.map` to do the same thing? If so, any reason to prefer one over the other?

Reply:
Gotcha. Yeah, the anonymous function passed into the filter is a little hard to read too